tests/shared/src/test/scala-2.12/scala/meta/tests/tokenizers/TokenizerCoverageSuite.scala

```diff
@@ -98,8 +98,11 @@ class TokenizerCoverageSuite() extends BaseTokenizerCoverageSuite {
   check[Term.Try]("try (→f←) catch { →case x => x;← →case y => y← } finally →{ }←")
   check[Term.Try]("try →()←")
   check[Term.Try]("try →(true, false)←")
+  check[Term.Try]("try →{1 + 2}.toString←")
```

Comment:
These arrows are to show what tokens are owned by which tree node. I can't remember the exact meaning of the arrows, but if we change how the parser encloses which tokens then these tests should fail so that we can better understand the implications.